manifests/init.pp

```diff
@@ -277,6 +277,8 @@
 #
 # $puppetca_certificate::       Token-whitelisting only: Certificate to use when encrypting tokens (undef to use SSL certificate)
 #
+# $registration_url::           URL that hosts will connect to when registering, defaults to the proxy URL.
```

Comment:
I'd say that it defaults to nothing. What Foreman then does with it is not the Smart Proxy's concern.

Reply:
Fair.